app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -518,7 +518,7 @@ private String prettyUploadedDate(Media media) {
         if (date == null || date.toString() == null || date.toString().isEmpty()) {
             return "Uploaded date not available";
         }
-        SimpleDateFormat formatter = new SimpleDateFormat("dd MMM yyyy", Locale.getDefault());
+        SimpleDateFormat formatter = new SimpleDateFormat("yyyy MMM dd", Locale.getDefault());
```

Comment:
We must find a solution that works in all languages.

Reply:
The problem was with the format of date right? Or have I misunderstood the issue.